src/ReactTestWrapper.js

```diff
@@ -29,7 +29,8 @@ export default class ReactTestWrapper extends TestWrapper {
   }
 
   attr (name) {
-    return this.el.getAttribute(name) || undefined
+    const value = this.el.getAttribute(name)
+    return value !== null ? value : undefined
```

Comment:
Yep, so there would be 3 cases:
- No attribute -> `undefined`
- empty attribute -> `true`
- attribute with valid value -> value
Am I right?